xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/ActionFilter.java

```diff
@@ -84,16 +86,25 @@ public void init(FilterConfig filterConfig) throws ServletException
     @SuppressWarnings("unchecked")
     @Override
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException,
-        ServletException
+            ServletException
     {
         // Only HTTP requests can be dispatched.
         if (request instanceof HttpServletRequest
-            && !Boolean.valueOf((String) request.getAttribute(ATTRIBUTE_ACTION_DISPATCHED))) {
+                && !Boolean.parseBoolean((String) request.getAttribute(ATTRIBUTE_ACTION_DISPATCHED)))
+        {
```

Comment:
Not XWiki style either.

Reply:
Same answer as above :)

Reply:
Actually it is Marius. When we have a condition that span more than 1 line the curly brace is on the next line to avoid the visual issue of the next line being at the same level.

Reply:
Was voted about 13 years ago now. I remember JV didn’t like it but in the end it was voted ;) I’ve been using ever since and fixing code when I saw it used wrongly.